src/server/call.ts

```diff
@@ -29,6 +29,19 @@ export class Call {
       throw new Error('Call info, method ID must be given');
     }
     let args: any = this.callInfo.bin_argument;
+    let metadata = new this.grpc.Metadata();
```

Comment:
I take it this metadata object isn't the same as the GRPC metadata message that gets sent from the client? we can't just avoid instantiating this and use that message instead?

Reply:
No, one is the object that grpc needs, the other is a `Map` object that is defined from a proto definition.

Reply:
Double checked ... they are in fact different types.